wordle-web/src/components/ThemeSelector.vue

```diff
@@ -0,0 +1,64 @@
+<template>
+  <v-dialog transition="dialog-bottom-transition" width="30rem">
+    <template v-slot:activator="{ props }">
+      <v-btn v-bind="props" class="ml-1" v-on:keydown.enter.prevent
+        ><v-icon icon="mdi-cog" size="x-large"></v-icon
```

Comment:
```suggestion
        > <v-icon icon="mdi-cog" size="x-large"> </v-icon
```
If you put spaces between these, your linter will format it a bit better.